src/middlewared/middlewared/etc_files/nsswitch.conf

```diff
@@ -23,8 +23,9 @@
         if ldap_enabled:
             ldap_anonymous_bind = safe_call('notifier.common', 'system', 'ldap_anonymous_bind')
             ldap_sudo_configured = safe_call('notifier.common', 'system', 'ldap_sudo_configured')
+            ldap = safe_call('datastore.query', 'directoryservice.LDAP')
 
-            if ldap_anonymous_bind:
+            if ldap_anonymous_bind and not ldap['ldap_kerberos_realm_id']:
```

Comment:
So if kerberos is specified, why are we not adding ldap here? What is the end goal here?

Reply:
Basically if we'll use sssd we're doing an anonymous bind with kerberos configured.